rusk-wallet/src/lib/prompt.rs

```diff
@@ -351,7 +351,7 @@ fn confirm(cmd: &CliCommand) -> bool {
             gas_limit,
             gas_price,
         } => {
-            let max_fee = gas_limit * gas_price.unwrap();
+            let max_fee = gas_limit.unwrap() * gas_price.unwrap();
```

Comment:
Should `unwrap_or_default` for a constant `MIN_GAS_LIMIT · 1.07`

Reply:
The option is **guaranteed** to have a `Some`